### PR TITLE
ENT-4012: Documentation about extensible menu was added.

### PR DESCRIPTION
**Pull request by aleksandrychev**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a pull request! Maybe @nickanderson can review this?

**Comment by aleksandrychev** [comment]
@nickanderson please review

File: guide/faq/extend-mission-portal-help-menu.markdown [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+Upload your html file into `/path/to/cfengine/httpd/htdocs/application/views/extraDocs/` directory 
+then your menu will be extended by this file. Name of menu item will be parsed from file name. Underscores will be replaced by whitespace. 
+Please note, uploaded files should have read permission for `cfapache` user.
+
+##Example
+
+File `test_documentation.html` was uploaded to the directory specified above.
+![Extended menu](extended-menu.png)
+
+##Html structure
```
Comment (nickanderson) [inline guide/faq/extend-mission-portal-help-menu.markdown]: I think I would call this Applying Mission Portal styling or something.

File: guide/faq/extend-mission-portal-help-menu.markdown [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Extend Mission portal help menu
```
Comment (nickanderson) [inline guide/faq/extend-mission-portal-help-menu.markdown]: I think I would put this together with the content about protected content. Both of them are related to "Extending Mission Portal". I would put them in the same page with dfferent headings for Adding Protected Content and Adding Custom user menu entries.

Comment (aleksandrychev) [reply guide/faq/extend-mission-portal-help-menu.markdown]: Yeah, good idea, honestly I wasn't sure about the place of these documentations.

**Pull request closed without merging**